cscs-checks/apps/gromacs/gromacs_check.py

```diff
@@ -113,49 +113,43 @@ def __init__(self, variant, **kwargs):
             self.num_tasks_per_node = 36
 
 
+@rfm.simple_test
 class GromacsCPUProdCheck(GromacsCPUCheck):
-    def __init__(self, **kwargs):
-        super().__init__('prod', **kwargs)
+    def __init__(self):
+        super().__init__('prod')
         self.tags |= {'production'}
         self.reference = {
             'dom:mc': {
-                'perf': (38.0, -0.05, None)
+                'perf': (42.7, -0.05, None)
             },
             'daint:mc': {
-                'perf': (73.0, -0.50, None)
+                'perf': (70.4, -0.20, None)
             },
         }
 
 
+@rfm.parameterized_test([(1,), (2,), (4,), (6,), (8,)])
```

Comment:
I have changed the decorator to accept multiple arguments, so that you can write this nicer as follows:

```python
@rfm.parameterized_test([1], [2], [4], [6], [8])
```

You can even use a range as follows:

```python
@rfm.parameterized_test([1], *([x] for x in range(2,9,2)))

```

Reply:
#294 is merged, so you can now use the syntax without the extra brackets.